src/Nri/Ui/Icon/V2.elm

```diff
@@ -533,6 +535,12 @@ greenCheckMark assets =
     ImgIcon assets.smallCheckmark_png
 
 
+{-| -}
+guidedWrite : { r | icons_guidedWrite_svg : Asset } -> IconType
+guidedWrite assets =
+    ImgIcon assets.icons_guidedWrite_svg
```

Comment:
❓ Looks like QuickWrite is also an ImgIcon... do you know what the pattern is for when something is supposed to be an image and when something is supposed to be an svg? I'm assuming that we want an image here because we want to use the given colors or something?

Reply:
That does sound like a good theory! TBH I'm not sure what the answer is (just copied from quick writes).

Will ask on Slack!